auth/schemas/permisos.ts

```diff
@@ -12,7 +12,11 @@ const schema = new mongoose.Schema({
             type: mongoose.Schema.Types.ObjectId,
             ref: 'organizacion'
         },
-        permisos: [String]
+        permisos: [String],
+        permisosPausados: {
```

Comment:
```suggestion
        active: {
```

Yo usaría un nobmre más básico y fácil para recordar.